app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -28,7 +28,7 @@ def new
       analytics.webauthn_setup_visit(**properties)
       save_challenge_in_session
       @exclude_credentials = exclude_credentials
-
+      @need_to_set_up_additional_mfa = in_multi_mfa_selection_flow? && mfa_selection_count < 2
```

Comment:
Should we include the `platform_authenticator` consideration here as well, and take it out of the view? Otherwise it feels odd to me that this value would be `true` for security keys as well.

Could maybe factor out a method if it helps deal with line length, e.g. `def need_to_set_up_additional_mfa?`